docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -0,0 +1,300 @@
+To disallow a namespace from being evaluated for mutation you must exclude it
+using the label `bindings.knative.dev/exclude: true`.
+
+In inclusion mode, SinkBinding behavior is not enabled for the namespace.
+Before a namespace can be evaluated for mutation you must
+explicitly include it using the label `bindings.knative.dev/include: true`.
+
+### Set to inclusion mode
+
+To set to inclusion mode, edit the `eventing-webhook` deployment to set
+`SINK_BINDING_SELECTION_MODE` to `inclusion`.
+The mode determines the default scope of the webhook.
+
+
+## Create a namespace
+
+If you do not have an existing namespace, create a namespace for the SinkBinding:
+
+```bash
+kubectl create namespace <namespace>
+```
+Where `<namespace>` is the namespace that you want your SinkBinding to use.
+For example, `sinkbinding-example`.
+
+**Note:** if you have selected inclusion mode, you must add the
+`bindings.knative.dev/include: true` label to the namespace to enable SinkBinding behavior.
+
+
+## Create an event sink
+
+The event sink can be any addressable Kubernetes object that can receive events.
+
+If you do not have an existing event sink that you want to connect to the SinkBinding,
+create a Knative service.
+
+=== "kn"
+
+    Create a Knative service by running:
+
+    ```bash
+    kn service create <app-name> --image <image-url>
+    ```
+    Where:
+    - `<app-name>` is the name of the application.
+    - `<image-url>` is the URL of the image container.
+
+    For example:
+
+    ```bash
+    $ kn service create hello --image gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
+    ```
+
+=== "YAML"
+    1. Copy the following YAML into a `service.yaml` file:
+        ```yaml
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        metadata:
+          name: <app-name>
+        spec:
+          template:
+            spec:
+              containers:
+                - image: <image-url>
+        ```
+        Where:
+        - `<app-name>` is the name of the application. For example, `event-display`.
+        - `<image-url>` is the URL of the image container.
+        For example, `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display`
+   </br></br>
+    2. Apply the YAML file by running:
+        ```bash
+        kubectl apply --filename service.yaml
+      ```
+
+
+
+
+## Create a subject
+
+The subject must be a PodSpecable resource.
+You can use any PodSpecable resource in your cluster, for example:
+
+- `Deployment`
+- `Job`
+- `DaemonSet`
+- `StatefulSet`
+- `Service.serving.knative.dev`
+
+If you do not have an existing PodSpecable subject that you want to use, you can
+use the following sample to create a `CronJob` object as the subject.
+The following cron job makes a single cloud event that targets `K_SINK` and adds
+any extra overrides given by `CE_OVERRIDES`.
+
+1. Copy the sample YAML below into a `cronjob.yaml` file:
```

Comment:
rewrite to use `kubectl apply -f - <<EOF` instead of copying into a YAML etc - see style guide